nemoguardrails/server/api.py

```diff
@@ -383,19 +558,33 @@ async def chat_completion(body: RequestBody, request: Request):
 
     try:
         llm_rails = _get_rails(config_ids)
+
     except ValueError as ex:
         log.exception(ex)
         return ResponseBody(
-            messages=[
-                {
-                    "role": "assistant",
-                    "content": f"Could not load the {config_ids} guardrails configuration. "
-                    f"An internal error has occurred.",
-                }
-            ]
+            id=f"chatcmpl-{uuid.uuid4()}",
```

Comment:
I think these sorts of transformation/conversions on request and response fits nicely to a utils module in schemas.

a simple function with
```
input:    response: GenerationResponse, config_ids=None
output: `GuardrailCompletionResponse`
```
same for streaming.

Also you can include [this logic](https://github.com/NVIDIA-NeMo/Guardrails/pull/1340/files#diff-e40b79fa06e87647988379efc0f1f3aa948eb153fdb73efc68989bdae0993d75R626-R644) there.


merely a suggestion, what do you think?